data/json/scenarios.json

```diff
@@ -527,5 +527,16 @@
     "allowed_locs": [ "mil_base_2g" ],
     "professions": [ "soldier", "specops", "bio_soldier", "bio_sniper" ],
     "flags": [ "CHALLENGE", "LONE_START" ]
+  },
+  {
+    "type": "scenario",
+    "ident": "presort",
+    "name": "Crazy party",
+    "points": 0,
+    "description": "Crazy party in secluded resort went very wrong - first cops arrived to ruin it, then dancers, guests and everyone around suddenly decided to have a piece of you.",
+    "start_name": "Private resort",
+    "allowed_locs": [ "p_resort_2ss" ],
+    "professions": [ "frat", "gangster", "lost_sub", "lawyer", "politician", "dancer" ],
```

Comment:
No Bionic Gangster?

Reply:
I missed it. Guess it would fit so I will add it with fixes that will most likely be neccessary.